inabox/deploy/deploy.go

```diff
@@ -265,6 +315,8 @@ func (env *Config) RegisterDisperserKeypair(ethClient common.EthClient) error {
 	return fmt.Errorf("timed out waiting for disperser address to be set")
 }
 
+// RegisterBlobVersionAndRelays initializes blob versions against ThresholdRegistry contract 
+// and relays against RelayRegistry contract
```

Comment:
```suggestion
// RegisterBlobVersionAndRelays initializes blob versions in ThresholdRegistry contract 
// and relays in RelayRegistry contract
```

Reply:
"against" sounds like its reading against onchain state to initialize offchain params